examples/scan_test.py

```diff
@@ -0,0 +1,110 @@
+"""
```

Comment:
This is cool but does it belong in this PR?

Reply:
not really - its a drive-by - would you like it elsewhere?
I am waiting for the actual code to come back from JS when he gets it done...